Pd 393 23.10 updates to storage articles (#2011)
truenas/documentation: content/SCALE/SCALEUIReference/Storage/Pools/PoolCreateWizardScreens.md

---
title: "Pool Creation Wizard Screens"
description: "Provides information on the settings and functions found on the Pool Creation Wizard screens."
weight: 30
aliases: 
tags: 
- scalepools
- scalestorage
- scalevdevs
- scaledisks
---

{{< toc >}}

The **Pool Creation Wizard** includes the wizard configuration screens, a configuration preview and inventory list of disks available on the system.

The **Create Pool (new)** button at the top right of the **Storage Dashboard** screen opens the **Pool Creation Wizard**. 

{{< trueimage src="/images/SCALE/23.10/PoolCreationWizard.png" alt="Pool Creation Wizard" id="1: Pool Creation Wizard" >}}

> The **Leave Feedback** link in the **Pool Creation Wizard** heading opens a feedback window that allows you to rate the web UI **Pool Creation Wizard** screen and leave comments. 

Author: I've created a subtask to address this part of the article after your pr for PD-391 merges.

{{< expand "Click Here for More Information" "v" >}}

{{< trueimage src="/images/SCALE/23.10/PoolCreationWizardGeneralInfo.png" alt="Pool Creation Wizard General Info" id="3: Pool Creation Wizard General Info" >}}

**Name** is a required field. 
Enter a name for the pool of up to 50 characters in length that follows [ZFS naming conventions](https://docs.oracle.com/cd/E23824_01/html/821-1448/gbcpt.html). 
Use lower-case alpha characters to avoid potential problems with sharing protocols. 
Names can include numbers and special characters such as underscore (_), hyphen (-), colon (:), or a period (.).. 

**Encryption** applies key type encryption to the pool. 
Select to enable [ZFS encryption](https://zfsonlinux.org/manpages/0.8.3/man8/zfs.8.html) for this pool and all datasets (or zvols) created within the pool. 
See [Storage Encryption]({{< relref "EncryptionScale.md" >}}) for more information on using SCALE storage encryption. An encryption warning dialog displays with a **Confirm** checkbox. Select to enable the **I Understand** button. **I Understand** allows you to continue adding the pool with encryption applied.

{{< hint type="Warning" >}}

> Applying encryption at the pool level also encrypts all datasets (and zvols) encrypted within the pool. 

Reviewer: ```suggestion
Applying encryption at the pool level also encrypts all datasets (and zvols) within the pool. 
```

> Keep the encryption key file in safe location where you perform regular backups. Losing the encryption key file results in lost data you cannot recover.

Reviewer: ```suggestion
Keep the encryption key file in a safe location where you perform regular backups. Losing the encryption key file results in lost data you cannot recover.
```

{{< /hint >}}

> If system disks have data exported from an pools on them, a warning displays with a checkbox for the pool name.

Reviewer: ```suggestion
If system disks have data exported from pools on them, a warning displays with a checkbox for the pool name.
```

{{< trueimage src="/images/SCALE/23.10/PoolCreationWizardGenInfoNonUniqueSerialNums.png" alt="Non-Unique Serial Numbers Screen" id="5: Non-Unique Serial Numbers Screen" >}}

**Allow non-unique serialed disks** has two radio buttons, **Allow** and **Don't Allow**.

**Allow** permits using disks with non-unique serial numbers, such as those that can occur on virtual disks, and displays them as available disks on the **Data** wizard screen. 
**Don't Allow** does not permit using disks with non-unique serial numbers.

{{< /expand >}}

### Data Wizard Screen
The **Data** wizard screen provides the option to add disks to the data VDEV. You must add a data VDEV before adding other types of VDEVs to the pool.

{{< trueimage src="/images/SCALE/23.10/PoolCreationWizardDataScreen.png" alt="Pool Creation Wizard Data Screen" id="6: Pool Creation Wizard Data Screen" >}}

{{< expand "Click Here for More Information" "v" >}}
**Layout** displays a dropdown list of VDEV layouts (**Stripe**, **Mirror**, **RAIDZ1**, **RAIDZ2**, and **RAIDZ3**). 

{{< include file="/_includes/PoolCreationWizardCommonSettings.md" type="page" >}}
{{< /expand >}}

### Manual Selection Screen

> The **Manual Selection** screen allows you to add a VDEV of the type specified in the **Layout** field (on the **Data** and **Log** wizard screens), and to select the disks to add to the new VDEV. You can filter disks by type or size. 

Reviewer: ```suggestion
The **Manual Selection** screen allows you to add a VDEV of the type specified in the **Layout** field (on the **Data** and **Log** wizard screens) and to select the disks to add to the new VDEV. You can filter disks by type or size. 
```

{{< expand "Click Here for More Information" "v" >}}
**Add** displays VDEV area of the type specified in the **Layout** field. 
Click on the system field to expand the dropdown list to show a list of available system disks. 
Drag the number of disks to VDEV to add them. 

**Save Selection** creates the VDEV and closes the window.

{{< trueimage src="/images/SCALE/23.10/PoolCreationWizardManualSelectionScreen.png" alt="Manual Selection Screen" id="6: Manual Selection Screen" >}}

#### Manual Selection Settings
{{< truetable >}}
| Setting | Description |
|---------|-------------|
| **Search** | Enter the disk name or other details to search for disks matching the specified value to filter available disks in the system. |
| **Filer by Disk Type** | Filters the list of available disks displayed on the list of system disks by type (HDD or SSD). |
| **Filter by Disk Size** | Filters the list of available disks displayed on the list of system disks by size. |

> | TrueNAS System | Click to expand and show the list of available disks in the system. Filter options change disks displayed on this list. |

Reviewer: ```suggestion
| **TrueNAS System** | Click to expand and show the list of available disks in the system. Filter options change disks displayed on this list. |
```

Author: TrueNAS System is not on the UI so it does not get bold emphasis.

{{< /expand >}}
### Cache (Optional) Wizard Screen
The **Cache** wizard screen provides the option to configure a ZFS L2ARC read-cache VDEV.

{{< trueimage src="/images/SCALE/23.10/PoolCreationWizardCacheScreen.png" alt="Pool Creation Wizard Cache Screen" id="9: Pool Creation Wizard Cache Screen" >}}

{{< expand "Click Here for More Information" "v" >}}

{{< include file="/_includes/PoolCreationWizardCommonSettings.md" type="page" >}}

{{< /expand >}}
### Metadata (Optional) Wizard Screen
The **Metadata** wizard screen provides the option to configure a special allocation class VDEV, for use when creating a fusion pool. This VDEV type is used to speed up metadata and small block IO.

{{< trueimage src="/images/SCALE/23.10/PoolCreationWizardMetadataScreen.png" alt="Pool Creation Wizard Metadata Screen" id="10: Pool Creation Wizard Metadata Screen" >}}

{{< expand "Click Here for More Information" "v" >}}

{{< include file="/_includes/PoolCreationWizardCommonSettings.md" type="page" >}}

{{< /expand >}}
### Dedup (Optional) Wizard Screen
The **Dedup** wizard screen provides the option to configure a VDEV to use for storing de-duplication tables. 

> Dedup VDEVs must be sized for *x* GiB for each *x* TiB of general storage.

Reviewer: ```suggestion
Dedup VDEVs must contain *x* GiB for each *x* TiB of general storage.
```
